### ISSUE TITLE
Does not run on files encoded in UTF-16

### ISSUE THREAD

**Issue by dturkelSalesforce** [lukejpreston/xunit-viewer#143]
* node and npm version - v20.4.0
* xunit viewer version - latest as installed 7/31/2023; "npx xunit-viewer --version" only returns "unknown"
* browser - N/A
* sample xml - See attached (renamed to ".txt")

The "tests.xml" file is generated via an Salesforce SFDX command to run unit tests, and write to file, then xunit-viewer is executed:
```
sfdx force:apex:test:run --codecoverage --detailedcoverage --resultformat junit --wait 100 > tests.xml
npx xunit-viewer -r tests.xml  
```
The result is:
`> npx xunit-viewer -r tests.xml
Failed to parse [omitted]\tests.xml
 Error: Non-whitespace before first tag.
Line: 0
Column: 1
Char: �

file:[omitted]/node_modules/xunit-viewer/src/cli/get-description.js:3
  Object.values(suites.suites).forEach((suite) => {
         ^

TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at default (file:[omitted]/node_modules/xunit-viewer/src/cli/get-description.js:3:10)
    at runXunitViewer ([omitted]/node_modules/xunit-viewer/xunit-viewer.js:26:25)
    at async default ([omitted]/node_modules/xunit-viewer/xunit-viewer.js:37:46)

Node.js v20.4.0`

I checked the file in multiple tools to ensure no non-visible characters were present - and there were none.
[tests.txt](https://github.com/lukejpreston/xunit-viewer/files/12218020/tests.txt)

Not a frequent node/npm user, so this could be user error -- would appreciate some guidance.

**Comment by lukejpreston** [comment]
Your file is saved as UTF-16 and if I save the file as UTF-8 it works

In VSCode you can change the encoding from UTF-16 to UTF-8 in the bottom right

From

<img width="103" alt="image" src="https://github.com/lukejpreston/xunit-viewer/assets/4129128/6186d58d-9443-4dc5-9869-7d8ee343426a">

To

<img width="103" alt="image" src="https://github.com/lukejpreston/xunit-viewer/assets/4129128/9589949b-6990-49c6-8fc3-ab71ba26d8b2">

I am not saying this is a permanent solution and I am looking into ways to normalise all files to UTF-8.
As a work around you can convert the files yourself and use Xunit Viewer.

I will change the title of this issue to reflect the problem and have a look as soon as possible 👍

**Comment by lukejpreston** [comment]
This should be resolved in `10.3.1`
